dashboard/lib/tasks/seed.rake

```diff
@@ -64,7 +64,7 @@ namespace :seed do
 
   # detect changes to dsldefined level files
   # LevelGroup must be last here so that LevelGroups are seeded after all levels that they can contain
-  DSL_TYPES = %w(TextMatch ContractMatch External Match Multi LevelGroup)
+  DSL_TYPES = %w(TextMatch ContractMatch External Match Multi EvaluationMulti LevelGroup)
```

Comment:
well if load_all_models doesn't load all models it probably should. It could actually look at the files itself (like the egrep line in the comment) instead of depending on people to add things to it.